charts/splice-helm/v0.1.5/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+version: 0.1.5
+appVersion: "3.0"
+name: splice-helm
+description: A Helm chart for launching Splice Machine on Kubernetes
+home: https://www.splicemachine.com
+icon: https://splicemachine.github.io/sm.logo.800x200px_180509.svg
```

Comment:
Please make a local copy of the logo in the parent folder `splice-helm/` and reference the image here as file://../splice-logo.png. This way the logo can be displayed in airtight environments.